apps/dynatrace/dynatrace-crds/kustomization.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - https://github.com/Dynatrace/dynatrace-operator/releases/download/v0.4.2/dynatrace.com_dynakubes.yaml
\ No newline at end of file
+  - https://github.com/Dynatrace/dynatrace-operator/releases/download/v0.4.2/dynatrace.com_dynakubes.yaml
+  #- https://github.com/Dynatrace/dynatrace-operator/releases/download/v0.9.1/dynatrace-operator-crd.yaml
```

Comment:
needed for new version but not really sure how to update this env by env so manually applied on env i'm testing on